test/cmd/help.sh

```diff
@@ -45,8 +45,8 @@ os::cmd::expect_success_and_text 'oc types' 'Deployment Config'
 os::cmd::expect_success_and_text 'openshift kubectl' 'Kubernetes cluster'
 os::cmd::expect_success_and_text 'oadm' 'Administrative Commands'
 os::cmd::expect_success_and_text 'openshift admin' 'Administrative Commands'
-os::cmd::expect_success_and_text 'oadm' 'Basic Commands:'
-os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
+os::cmd::expect_success_and_text 'oadm' 'Component Installation:'
```

Comment:
Drain, Taint, and Cordon were not exposed.  I exposed them, then
reorganized.

On Wed, Jul 6, 2016 at 10:01 AM, David Eads <notifications@github.com>
wrote:

> In test/cmd/help.sh
> <https://github.com/openshift/origin/pull/9578#discussion_r69735211>:
>
> > @@ -45,8 +45,8 @@ os::cmd::expect_success_and_text 'oc types' 'Deployment Config'
> >  os::cmd::expect_success_and_text 'openshift kubectl' 'Kubernetes cluster'
> >  os::cmd::expect_success_and_text 'oadm' 'Administrative Commands'
> >  os::cmd::expect_success_and_text 'openshift admin' 'Administrative Commands'
> > -os::cmd::expect_success_and_text 'oadm' 'Basic Commands:'
> > -os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
> > +os::cmd::expect_success_and_text 'oadm' 'Component Installation:'
>
> What does this have to do with the rebase?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/dacd357f9121e6ad252cb0c5931cc4720a1ddea6..993c4ac1fd086af40ae0e0c811ce9052ce185714#r69735211>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p2jmqCFNvqjR6mf5Dd9QcH2xuS_vks5qS7UrgaJpZM4I-xTE>
> .
>